opensearch-operator/api/v1/opensearch_types.go

```diff
@@ -43,7 +43,8 @@ type GeneralConfig struct {
 	// Extra items to add to the opensearch.yml
 	AdditionalConfig map[string]string `json:"additionalConfig,omitempty"`
 	// Drain data nodes controls whether to drain data notes on rolling restart operations
-	DrainDataNodes bool `json:"drainDataNodes,omitempty"`
+	DrainDataNodes bool     `json:"drainDataNodes,omitempty"`
```

Comment:
unnecessary formatting change? the line above (44) is also not aligned with this